### PR TITLE
Fix stability PD Failover check

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
Fix the pd failover check in stability test

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: tests/failover.go [hunk 1]
```diff
@@ -61,43 +61,74 @@ func (oa *operatorActions) DeletePDDataThenCheckFailover(info *TidbClusterConfig
 	}
 	klog.Infof("delete pod %s/%s data successfully", ns, podName)
 
+	oldPD, err := oa.kubeCli.CoreV1().Pods(ns).Get(podName, metav1.GetOptions{})
+	if err != nil {
+		klog.Error(err.Error())
```
Comment (zjj2wry) [inline tests/failover.go:66]: klog.Error(err)

Comment (Yisaer) [reply tests/failover.go]: updated.

**Pull request merged**